graphql/src/execution/execution.rs

```diff
@@ -89,16 +88,45 @@ struct HashableQuery<'a> {
 /// different representations. This is considered not an issue. The worst
 /// possible outcome is that the same query will have multiple cache entries.
 /// But, the wrong result should not be served.
+impl stable_hash_legacy::StableHash for HashableQuery<'_> {
```

Comment:
I don't think we need the legacy version for this type since it's not used in the PoI. We can of course keep it around, but if you want to find opportunities to remove bloat this is one. There are probably some other types this applies to as well.